services/wallet/bridge/mock_bridge/bridge.go

```diff
@@ -8,9 +8,10 @@ import (
 	big "math/big"
 	reflect "reflect"
 
+	gomock "github.com/golang/mock/gomock"
+
 	common "github.com/ethereum/go-ethereum/common"
 	types "github.com/ethereum/go-ethereum/core/types"
-	gomock "github.com/golang/mock/gomock"
```

Comment:
This looks like the result of an auto-import after initially removing the code that depended on the import.